src/content/docs/guides/Beginners/Your First Shader/1_composite.mdx

```diff
@@ -0,0 +1,167 @@
+---
+title: Simple Post Effects
+description: Write a basic post processing effect in the `composite` pass.
+sidebar:
+  label: Simple Post Effects
+  order: 2
+---
+
+## Setting up the file structure
+Minecraft shaders require a specific structure of files in the right places to load code. While it's important to understand this structure, to save time, we will be working with the Base 330 pack from shaderLABs. Download it from [here](https://github.com/shaderLABS/Base-330), and extract it into your `shaderpacks` folder. You should have the following structure.
+
+```
+.
+└── .minecraft/
+    └── shaderpacks/
+        └── Base-330-main/
+            ├── LICENSE
+            ├── readme.md
+            └── shaders/
+                └── ...
+```
+
+In this stage of the tutorial, we are going to make the screen monochrome using the `composite` pass, so let's rename the `Base-330-main` folder to `composite-tutorial`, or another name of your choosing.
+
+You can also delete the `LICENSE` and `readme.md` files, as we don't need those.
+
+:::caution[Note]
+It is always important to respect the license associated with code when you use it anywhere. In this case, the base pack is licensed under [The Unlicense](https://choosealicense.com/licenses/unlicense/), meaning you can do what you want with it, however other shaders may use more restrictive licenses for their code.
+:::
+
+When you select the shader in the shader selection screen, you should not see any errors in the logs.
+
+## The `composite` pass
+For this shader, we will be using the first `composite` pass. This is a full screen pass which runs just after all gbuffers programs have rendered.
+
+First, let's open `composite.vsh`. This is the *vertex shader* for the `composite` program. Since `composite` is a fullscreen pass, this actually just renders a singular quad to the screen which exactly covers it. This means that your fullscreen passes are technically actually running on 3D geometry!
```

Comment:
First use of quad, explain if possible (I can make a glossary page if you want at some time)

Reply:
Not sure a glossary is necessary. Can add context that a quad is a plane and I think that's as much user clarification as we want to provide since a basic knowledge of linear algebra is assumed.